src/dependencies.ts

```diff
@@ -51,3 +52,15 @@ try {
   relayConfigMod = require("relay-config")
 } catch (_) {}
 export const RelayConfig = relayConfigMod
+
+let relayCompilerMainMod: typeof _RelayCompilerMain | null = null
+try {
+  // relay-compiler v6
+  relayCompilerMainMod = require("relay-compiler/bin/RelayCompilerMain")
+} catch (_) {
+  try {
+    // relay-compiler v5
+    relayCompilerMainMod = require("relay-compiler/lib/RelayCompilerMain")
+  } catch (_) {}
```

Comment:
Should we catch this error and state that we could not find a config at all?

Reply:
Can do 👍